gradle.properties

```diff
@@ -12,7 +12,6 @@ org.jetbrains.dokka.kotlinLanguageLevel=1.4
 
 # Build Scans are disabled by default - see CONTRIBUTING.md to opt in
 org.jetbrains.dokka.build.scan.url=https\://ge.jetbrains.com/
```

Comment:
while we are here and we removed `cache.url`, what is the reason to have `scan.url` here and not inside settings plugin (like with `cache.url`)?

Reply:
Good point, I'll hardcode the build scan url too.

The reason the scan url is configurable is because originally I intended that users could opt-out by not defining the url, but that didn't work out so well (I can't remember why) so I added an explicit boolean opt-in/out, but didn't update the scan url.